Buildings/HeatTransfer/Data/OpaqueConstructions.mo

```diff
@@ -11,7 +11,8 @@ package OpaqueConstructions
       annotation (choicesAllMatching=true, Evaluate=false, Placement(transformation(extent={{60,60},{80,80}})));
    final parameter Real R(unit="m2.K/W")=sum(material[i].R for i in 1:nLay)
       "Thermal resistance per unit area";
-
+   parameter Integer nSta2[nLay](each min=1)={material[i].nSta for i in 1:nLay}
```

Comment:
I couldn't come up with an alternative . We could add a comment along the line "(do not overwrite, used to work around Dymola 2017 bug)"...